Makefile

```diff
@@ -1,6 +1,6 @@
 PROJECT = hut
 PROJECT_DESCRIPTION = helper library for making Erlang libraries logging framework agnostic
-PROJECT_VERSION = 1.3.0
+PROJECT_VERSION = 1.3.1
```

Comment:
Don't increment the version. This should only be done when releasing.

Reply:
Rolled back